scripts/ini_template_generator.py

```diff
@@ -0,0 +1,221 @@
+#                                        If given, it is assumed you want grain size output.
+#                                        Additionally, if this mode is selected, the supplied files must be .LAS with extra bytes included with
+#                                        "Intensity as Reflectance" returned by RIEGL.
+#     reflectance.imu_data ........... (optional): path to helicopter IMU .CSV or.TXT data used to match data with point cloud using GPS time.
+#                                        Column names must include ['Time[s]', 'Easting[m]', 'Northing[m]', 'Height[m]'] 
+#     reflectance.cal_las ............ (optional): path to .LAS used for calibration of the apparent reflectance for 1064nm of lidar sensor.
+#                                        To avoid confusion, please supply this file in a different directory from <in_dir>.
+#     reflectance.known_rfl .......... (optional, float/real): known intrinsic reflectance at 1064 nm for target identified in shp_fp_rfl
+#     reflectance.h2o ................ (optional, float): water column vapor in atmosphere (mm)
+#     reflectance.aod ................ (optional, float): aerosol optical depth at 550 nm
+#
+#     smrf.scalar .................... (optional, float): SMRF scalar parameter override
+#     smrf.slope ..................... (optional, float): SMRF slope override
+#     smrf.threshold ................. (optional, float): SMRF elevation threshold override
+#     smrf.window .................... (optional, float): SMRF window size override
+# ==============================================================================================================================================
+
+[general]
+input_dir = <required>
+debug = false
+
+[dem]
+user_dem =
+is_geoid = <required if user_dem>
+
+[alignment]
+shapefile = <required>
+buffer_meters = 3.0
+asp_dir =
+
+[reflectance]
+shp_fp_rfl =
+imu_data =
+cal_las =
+known_rfl =
+h2o =
+aod =
+
+[smrf]
+scalar =
+slope =
+threshold =
+window = 
+"""
+
+# ----------------------------------------------------------------------
+# Logging setup
+# ----------------------------------------------------------------------
+
+def setup_logging(enable_file: bool = True) -> logging.Logger:
+    """
+    Configure logging for ini_template_generator.
+
+    - Logs to both console and (optionally) a timestamped file in ./logs/
+    - File logging can be disabled with enable_file=False
+    """
+    
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.INFO)
+
+    # Prevent duplicate handlers on repeated runs
+    if log.hasHandlers():
+        return log
+
+    # Console handler (always active)
+    console_handler = logging.StreamHandler()
+    console_handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
+    log.addHandler(console_handler)
+
+    if enable_file:
+        log_dir = Path.cwd() / "logs"
+        log_dir.mkdir(exist_ok=True)
+
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Comment:
Since this is a log file Can we make the format a bit more user friendly and remove the underscore between date and time?